### DISCUSSION TITLE
Season Disruption

### DISCUSSION THREAD

**Discussion by kamikazecanuck** [110yards/fantasy#152] [General]
So how bad would a delayed start to the season mess up the site?  Say I've already generated my schedule and completed my draft but the the season's start is delayed a few weeks?

Would we have to regenerate the schedule and redraft?

**Comment by mdryden** [comment]
I'm not totally sure, we might have to wait and see.

In theory, I just need to update the schedule and as long as the number of weeks haven't changed, everything should work.  Nothing in the site is hard coded to use specific dates or anything.

The only thing I can think of that would cause problems is if the season is shortened, then there is the chance that some leagues might have an invalid playoff start, since it's all based on the number of weeks in the season, which I have to set ahead of time each year.

Once we have confirmation from the league I'll make whatever changes are necessary.

**Comment by kamikazecanuck** [comment]
Ok thanks. Hopefully they can get their act together.
